cpan/Pod-Perldoc/corpus/perlfunc.pod

```diff
@@ -6423,11 +6423,12 @@ how many places right of the decimal point to show (the default being 6).
 For example:
 
   # these examples are subject to system-specific variation
-  printf '<%f>', 1;    # prints "<1.000000>"
-  printf '<%.1f>', 1;  # prints "<1.0>"
-  printf '<%.0f>', 1;  # prints "<1>"
-  printf '<%e>', 10;   # prints "<1.000000e+01>"
-  printf '<%.1e>', 10; # prints "<1.0e+01>"
+  printf '<%f>', 1;       # prints "<1.000000>"
+  printf '<%.1f>', 1;     # prints "<1.0>"
+  printf '<%.0f>', 1;     # prints "<1>"
+  printf '<%07.2f>', 1.3; # prints "<0001.30>"
+  printf '<%e>', 10;      # prints "<1.000000e+01>"
+  printf '<%.1e>', 10;    # prints "<1.0e+01>"
```

Comment:
This file isn't documentation and isn't maintained by p5p (and doesn't match the core perlfunc.pod)

Reply:
What can I do here?

Reply:
Don't change it.  It is used as a test file by Pod::Perldoc, not as documentation.

Reply:
Thanks.

Hopefully #21144 will make this less confusing.